Proven-In-Use-Argument.md

```diff
@@ -0,0 +1,77 @@
+# Is Linux mature and proven enough for use in safety-critical applications?
+
+Linux has proven itself thousands, probably millions of times over in business-critical applications. So it makes sense to use Linux for safety-critical applications as well. Linux has many advantages 
+
+* it runs on an enormously wide range of hardware
+* there are a large number of established vendors and service providers for the operating system and for additional (standard) software
+* many specialists are familiar with it
+* it is continuously being developed
+* a large part of the innovation in IT is created on Linux and as open source software
+* ...
+
+Linux has obviously proven itself. So what is so difficult about safety-critical applications?
+
+First, it's useful to look briefly at what functional safety is actually about. At its core, it is about dangerous items: technical systems, devices, machines, vehicles and the like. Such items can harm the environment or people, and in the worst case can even kill people uninvolved in their operation.
```

Comment:
I think we should consider using 'product' or 'products' instead of 'item' or 'items' in this and other sentences. The word 'item' has a specific meaning in ISO 26262, which may cause readers to think that we are using the word in that sense in some circumstances. 

I would also make the following change:

```suggestion
First, it's useful to look briefly at what functional safety is actually about. At its core, it is about potentially dangerous items: technical systems, devices, machines, vehicles and the like, which can harm the environment or people, and in the worst case can even kill people uninvolved in their operation.
```

Reply:
I see the problem with the term item although it does intentionally lead into the direction of its use in ISO 26262. Product appears to soft for me. What do you think about thing?

Reply:
Upon further consideration, I'm happy with item